app/src/main/java/com/odysee/app/MainActivity.java

```diff
@@ -1267,18 +1267,11 @@ public void openFileUrl(String url, String source) {
     }
 
     private void openSpecialUrl(String url, String source) {
-        String specialPath = url.substring(8).toLowerCase();
-        if (specialRouteFragmentClassMap.containsKey(specialPath)) {
-            Class fragmentClass = specialRouteFragmentClassMap.get(specialPath);
-            if (fragmentClassNavIdMap.containsKey(fragmentClass)) {
-                Map<String, Object> params = null;
-                if (!Helper.isNullOrEmpty(source)) {
-                    params = new HashMap<>();
-                    params.put("source",  source);
-                }
-
-//                openFragment(specialRouteFragmentClassMap.get(specialPath), true, fragmentClassNavIdMap.get(fragmentClass), params);
+        if (url.equalsIgnoreCase("lbry://?rewards")) {
```

Comment:
There are other special URLs that may be used, which is why we need to maintain the `specialRouteFragmentClassMap`. Is there a specific reason for removing the hashmap?